src/App.js

```diff
@@ -1,16 +1,21 @@
 import './App.css';
 import  Form  from "./Components/Form"
-import { useState } from "react"
+import List from "./Components/List"
```

Comment:
```suggestion
import List from "./components/List"
```
Just a really tiny hint: in most projects you would see the `components` folder without capitalization. But for subfolders (the real components) it's a good convention.